packages/common/src/types/db/types.ts

```diff
@@ -34,6 +34,15 @@ export const tags = [
   "solo",
 ] as const;
 
+export const editableFeeRecipientTags = [
+  "obol",
+  "diva",
+  "ssv",
+  "stakewise",
+  "stakehouse",
```

Comment:
This is not editable

Reply:
I know, but for the moment we are not handling the fee recipient automatically, so I left them as editable